lib/simulation/openfisca/mapping/index.js

```diff
@@ -101,23 +75,24 @@ var applyRessources = function(mesAidesEntity, openfiscaEntity, mappingSchema, s
                     result[periods.thisMonth.key] += fn ? fn(ressourceLastMonth.montant) : ressourceLastMonth.montant;
                 }
 
-                openfiscaEntity[openfiscaKey] = result;
-
                 // On suppose que les revenus pour l'année fiscale de référence sont les mêmes que pour les 12 derniers mois, si ces revenus n'ont pas encore
                 // été renseignés
-                var rollingYearKey = periods.rollingYear.key,
-                    anneeFiscaleReferenceKey = periods.anneeFiscaleReference.key;
+                var anneeFiscaleReferenceKey = periods.anneeFiscaleReference.key;
                 if (! situation.ressourcesYearMoins2Captured) {
-                    openfiscaEntity[openfiscaKey][anneeFiscaleReferenceKey] = openfiscaEntity[openfiscaKey][rollingYearKey];
+                    result[anneeFiscaleReferenceKey] = periods.last12Months.reduce(function(sum, periodObject) {
+                        return sum + result[periodObject.key];
```

Comment:
No idea, I don't see understand my own comment 😞

There is no “correct” indent but I personally prefer version 1.